pkg/machine/gvproxy_windows.go

```diff
@@ -0,0 +1,42 @@
+package machine
+
+import (
+	"os"
+	"time"
+
+	"github.com/containers/winquit/pkg/winquit"
+	"github.com/sirupsen/logrus"
+)
+
+func waitOnProcess(processID int) error {
+	logrus.Infof("Going to stop gvproxy (PID %d)", processID)
+
+	p, err := os.FindProcess(processID)
+	if err != nil {
+		return nil
```

Comment:
I assume this is intentional? If so this needs a comment because it really looks like a typo and should be error.

Reply:
that's right this is intentional. the contract of find process on windows is to return an error when the process is not found, and the goal of the method is to make sure the process is no longer there. Ill add a comment.